bundle-workflow/src/test_workflow/transform_MensorResults.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import json
+
+import json2table
+
+
+class JsonTransformation:
+
+    def convert_to_html(self, json_file, path_to_jsonfile):
+        with open(path_to_jsonfile + '/' + json_file) as file:
+            try:
+                infoFromJson = json.load(file)
+                testExecutionId = infoFromJson['testExecutionId']
+                build_direction = "TOP_TO_BOTTOM"
+                table_attributes = {"style": "width:100%", "border": "solid"}
```

Comment:
Can you provide a screenshot of how this renders?

Reply:
<img width="861" alt="Screen Shot 2021-09-02 at 9 56 48 AM" src="https://user-images.githubusercontent.com/44311835/131885822-cce81740-7e19-4945-bfbb-21e46d300028.png">

This is for the vertical view


<img width="1792" alt="Screen Shot 2021-09-02 at 10 00 04 AM" src="https://user-images.githubusercontent.com/44311835/131886200-95ac5d06-e7ee-4419-9ba8-991b6a1ac7b6.png">

This is for the horizontal view